CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (modules/core/04-channel) [\#1232](https://github.com/cosmos/ibc-go/pull/1232) Updating params on `NewPacketId` and moving to bottom of file. 
 * (modules/core/04-channel) [\#1279](https://github.com/cosmos/ibc-go/pull/1279) Add selected channel version to MsgChanOpenInitResponse and MsgChanOpenTryResponse. Emit channel version during OpenInit/OpenTry
 * (app/29-fee) [\#1305](https://github.com/cosmos/ibc-go/pull/1305) Change version string for fee module to `ics29-1`
+* (app/29-fee) [\#1340](https://github.com/cosmos/ibc-go/pull/1340) Return error if fee module is locked before distributing fees on acknowledgement and timeout
```

Comment:
We can remove this entry since we haven't released ics29 yet (and this is just fixing an internal bug)